setup.cfg

```diff
@@ -0,0 +1,38 @@
+[metadata]
+name = pytest-marker-bugzilla
+author = Eric Sammons
+author-email = elsammons@gmail.com
+summary = py.test bugzilla integration plugin, using markers
+description-file = README.md
+home-page = https://github.com/eanxgeek/pytest_marker_bugzilla.git
+license = GPLv2
+zip_safe=False
+entry_points={
+        'pytest11': ['pytest_marker_bugzilla = pytest_marker_bugzilla'],
+    }
+classifier =
+    Development Status :: 3 - Alpha
+    License :: OSI Approved :: GNU General Public License 2 (GPLv2),
```

Comment:
still wrong, you can just copy it ...

`License :: OSI Approved :: GNU General Public License v2 (GPLv2)`